int32 id
float64[8] channels_data
string board_type
time start_time
geometry_msgs/Vector3 aux_data